cli/command/trust/signer-remove.go

```diff
@@ -0,0 +1,54 @@
+package trust
+
+import (
+	//"encoding/hex"
+	//"fmt"
+	//"sort"
+	//"strings"
+
+	//"github.com/Sirupsen/logrus"
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	//"github.com/docker/cli/cli/command/formatter"
+	"github.com/docker/cli/cli/trust"
+	//"github.com/docker/notary"
+	//"github.com/docker/notary/client"
+	"github.com/docker/notary/tuf/data"
+	"github.com/spf13/cobra"
+)
+
+func newSignerRemoveCommand(dockerCli command.Cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "signer-remove IMAGE NAME",
```

Comment:
we should make this command work across multiple images. Perhaps it changes to:

`signer-remove [OPTIONS] NAME IMAGE [IMAGE...]` ?